applications/rag/frontend/container/main.py

```diff
@@ -160,7 +166,10 @@ def handlePrompt():
             "context": context,
             "user_prompt": user_prompt
         })
-        if 'inspectTemplate' in data and 'deidentifyTemplate' in data:
+        if 'nlpFilterLevel' in data:
+            if nlp_filter.is_content_inappropriate(response['text'], data.get['nlpFilterLevel']):
```

Comment:
What is the default for this nlpFilterLevel?

Reply:
50, defined in the html